packages/plugin-core/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": [
+    "src/**/*.ts"
+  ], 
+  "compilerOptions": {
+    "rootDir": "src", 
+    "allowSyntheticDefaultImports": true, 
+    "target": "ES2015",
+		"declaration": true,
+		"declarationDir": "dist/dts",
```

Comment:
I'm curious to know why you had to set this here 🤔

Reply:
I used `rollup-plugin-dts` to roll all the `.d.ts` files into one file here: https://github.com/honeybadger-io/honeybadger-js/blob/4c084a21d8b646817b39b1025af21bea9211def1/packages/plugin-core/rollup.config.mjs#L20-L21 and then used `rollup-plugin-delete` to get rid of the rest of the `.d.ts` files in the `dts` folder. This is not strictly necessary, since having multiple `.d.ts` files works fine, but makes the bundle nice and tidy with only `index.js`, `index.d.ts`, and `index.js.map`. 

As I learn more about rollup, I'm starting to appreciate making a smaller, tidier bundle... I don't know if we want to choose some kind of best practice standard for bundling but doing it this way was definitely a good learning experience.

Reply:
OK sounds good, I think I will give it a go at some point!